js/rpg_core/Utils.js

```diff
@@ -125,3 +125,8 @@ Utils.rgbToCssColor = function(r, g, b) {
     b = Math.round(b);
     return 'rgb(' + r + ',' + g + ',' + b + ')';
 };
+
+Utils._id = 1;
+Utils.generateRuntimeId = function(){
+    return Utils._id++;
```

Comment:
It doesn't matter.
rotating id makes another problem.
We should not implement this strictly.